order_test.go

```diff
@@ -132,3 +132,19 @@ func TestListOrders(t *testing.T) {
 		}
 	}
 }
+
+func TestCancelAll(t *testing.T) {
+	client := NewTestClient()
+
```

Comment:
Won't this always fail if it's run without any outstanding orders?

Reply:
I'm not sure. :) 

Not very experienced with Go, and with the public sandbox being down I couldn't actually run the test suite. Seems to still be down right now.

I'll add some glue to ensure it always has an order to cancel anyway.

Reply:
I think if you make an order with a really high size, that should be sufficient